packages/site-server/src/catalog/router.ts

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import Router from '@koa/router';
+
+export const catalogRouter = new Router();
+
+catalogRouter.get('/element/:path+', async (context) => {
+  const {params} = context;
+
+  // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
I'm finding this rule to be pretty annoying and of questionable value. There are lots of places like this where the code can't reach unless this expression is non-null (in the case the pattern matching done by `.get()`, later in the file the pattern matching done by `replaceAll()`) and I'd rather not add optional-chaining or other checks just to satisfy the type-checker.

Reply:
Sure, I don't mind removing it.